src/lib.rs

```diff
@@ -417,40 +447,41 @@ impl HexView {
         for lines in 0..self.get_widget_height() {
             printer.print(
                 (0, lines),
-                &format!("{:0len$X}", lines * U8S_PER_LINE, len = digits_len),
+                &format!("{:0len$X}", lines *  self.config.bytes_per_line, len = digits_len),
             );
         }
     }
 
     fn draw_addr_hex_sep(&self, printer: &Printer) {
-        printer.print_vline((0, 0), self.get_widget_height(), ADDR_HEX_SEPERATOR);
+        printer.print_vline((0, 0), self.get_widget_height(),  &self.config.addr_hex_separator);
     }
 
     /// draws the hex fields between the addr and ascii representation
     fn draw_hex(&self, printer: &Printer) {
-        for (i, c) in self.data.chunks(U8S_PER_LINE).enumerate() {
+        let spacer = &format!("{:width$}", "", width=self.config.chars_per_spacing);
```

Comment:
Please use https://doc.rust-lang.org/std/primitive.str.html#method.repeat instead, e.g.

`" ".repeat(self.config.chars_per_spacing)`

Reply:
Let me know if [this](https://github.com/palako/cursive_hexview/commit/5b7ddb3fc73daa4ee163bb7cd5fe0caac2efb29a) is good

Reply:
I would probably put the `&` when using `spacer` and not while declaring it, but yes (: